Assets/_BForBoss/_Utility/Scripts/Logger/Logger.cs

```diff
@@ -2,14 +2,23 @@
 
 namespace Perigon.Utility
 {
+    public enum LoggerColor
+    {
+        Black,
+        Red,
+        Blue,
+        Green,
+        Yellow
+    }
+    
     public static partial class Logger
     {
-        public static void LogString(string toLog, string key = "Misc")
+        public static void LogString(string toLog, LoggerColor color = LoggerColor.Black,  string key = "Misc")
         {
 #if !UNITY_EDITOR && DEBUG
-            Debug.Log(toLog);
+            Debug.Log(ColorizeLog(toLog, color));
 #elif UNITY_EDITOR
-            LogInEditor(toLog, key, Debug.Log);
+            LogInEditor(ColorizeLog(toLog, color), key, Debug.Log);
 #endif
         }
         
```

Comment:
Decided to not add the parameter to our LogFormat method since that method will already be doing intentional special formatting to its text, so it seemed unnecessary to add